collector/internal/collector/collector.go

```diff
@@ -33,8 +33,13 @@ import (
 	"go.uber.org/zap/zapcore"
 )
 
-// Collector implements the OtelcolRunner interfaces running a single otelcol as a go routine within the
-// same process as the test executor.
+type CollectorWrapper interface {
```

Comment:
Should this interface go in the `lifecycle` package since that's where it is used?

Reply:
makes sense, moved.